### ISSUE TITLE
[BUG/ISSUE] xbpch cannot open ND51 bpch output

### ISSUE THREAD

**Issue by 1Dandan** [geoschem/gcpy#123]
## Describe the bug
I am trying to convert bpch file to netCDF file, but a simple test on xbpch reporting attribute conflicts with xarray.
The error message is:
![error_message](https://user-images.githubusercontent.com/75513026/118564893-cf5c9400-b736-11eb-8350-91e605af5bb0.PNG)

## To Reproduce
The problem can be reproduced by the python code below and attachments:
[attachments.zip](https://github.com/geoschem/gcpy/files/6497192/attachments.zip).

```python
import xarray as xr
import xbpch as xb

bpchfile = 'ts_10_12.20190101.bpch'
tinfo_file = 'tracerinfo.dat'
dinfo_file = 'diaginfo.dat'

ncfile = 'ts_10_12.20190101.nc4'

ds = xb.open_bpchdataset(filename=bpchfile, \
    tracerinfo_file=tinfo_file, \
    diaginfo_file=dinfo_file)
```

## Required information:
It says gcpy has no attribute of show_versions, but by conda list, it showed the geoschem-gcpy version is 1.0.3
Run the following commands from an interactive Python session:
```python
>>> import gcpy
>>> gcpy.show_versions()
```
```
AttributeError                            Traceback (most recent call last)
<ipython-input-2-751f1da69d2a> in <module>
----> 1 gcpy.show_versions()

AttributeError: module 'gcpy' has no attribute 'show_versions'
```

**Comment by yantosca** [comment]
Thanks for writing @1Dandan.  You might have to use an older version of xarray for xbpch to work.  Xbpch is no longer being actively maintained (because the developer has since left the GEOS-Chem community) and I think some recent modifications to xarray break the current xbpch.

I think you need xarray 0.11 for xbpch to work.  You should be able to revert the version in conda with
```
conda install xarray=0.11
```

**Comment by 1Dandan** [comment]
Thank you for following up. The bpch file reading problem with xbpch is solved with python = 3.7, xarray = 0.11.
